<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>604</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>604</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openMyComBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeMyComBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 50, 14);</string>
    </property>
    <property name="text">
     <string>当前温度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>当前湿度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_liang">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_mie">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/22.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>641</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/主界面.jpg);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="green">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/绿灯.jpg);
background-color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="red">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/红灯.jpg);
background-color: rgb(213, 213, 213);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>80</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>温度设置</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);</string>
     </property>
     <property name="text">
      <string>温度上限：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);</string>
     </property>
     <property name="text">
      <string>温度下限：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ceil">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="floor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 255);
background-color: rgb(0, 0, 0);
background-color: rgb(235, 232, 232);</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>openMyComBtn</zorder>
   <zorder>closeMyComBtn</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_liang</zorder>
   <zorder>label_mie</zorder>
   <zorder>green</zorder>
   <zorder>red</zorder>
   <zorder>groupBox</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>lcdNumber_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
